frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -148,46 +105,69 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
 type SelectCapacityAndNodesProps = {
   dispatch: WizardDispatch;
   capacity: WizardState['capacityAndNodes']['capacity'];
-  nodes: WizardState['capacityAndNodes']['nodes'];
+  nodes: WizardState['nodes'];
 };
 
 const SelectedCapacityAndNodes: React.FC<SelectedCapacityAndNodesProps> = ({
   capacity,
   storageClassName,
   enableArbiter,
+  arbiterLocation,
   dispatch,
   nodes,
 }) => {
   const { t } = useTranslation();
   const [pv, pvLoaded, pvLoadError] = useK8sWatchResource<K8sResourceKind[]>(pvResource);
   const memoizedPv = useDeepCompareMemoize(pv);
   const [allNodes, allNodeLoaded, allNodeLoadError] = useK8sWatchResource<NodeKind[]>(nodeResource);
-  const memoizedNodes = useDeepCompareMemoize(allNodes);
+  const memoizedAllNodes = useDeepCompareMemoize(allNodes);
+
+  const pvBySc = React.useMemo(() => getSCAvailablePVs(memoizedPv, storageClassName), [
+    memoizedPv,
+    storageClassName,
+  ]);
 
   React.useEffect(() => {
-    if (pvLoaded && !pvLoadError && memoizedPv.length) {
-      const pvBySc = getSCAvailablePVs(memoizedPv, storageClassName);
+    if (pvLoaded && !pvLoadError) {
       const pvCapacity = calcPVsCapacity(pvBySc);
       const humanizedCapacity = humanizeBinaryBytes(pvCapacity).string;
-      dispatch({ type: 'capacityAndNodes/capacity', payload: humanizedCapacity });
-      dispatch({ type: 'capacityAndNodes/pvCount', payload: memoizedPv.length });
+      dispatch({
+        type: 'capacityAndNodes/capacity',
+        payload: humanizedCapacity,
+      });
+      dispatch({ type: 'capacityAndNodes/pvCount', payload: pvBySc.length });
     }
-  }, [dispatch, memoizedPv, pvLoadError, pvLoaded, storageClassName]);
+  }, [dispatch, pvBySc, pvLoadError, pvLoaded, t]);
```

Comment:
```suggestion
  }, [dispatch, pvBySc, pvLoadError, pvLoaded]);
```
Where is it being used?